test/subarray.jl

```diff
@@ -479,7 +479,7 @@ Y = 4:-1:1
 
 @test isa(@view(X[1:3]), SubArray)
 
-@test X[1:end] == @view X[1:end]
+@test X[1:end] == @. @view X[1:end]
```

Comment:
Was your intention here to exercise the interaction of `@.` with `@view`? If so, perhaps add a comment to that effect?